shared/reducers/index.js

```diff
@@ -6,6 +6,7 @@ import devEdit from './dev-edit'
 import devices from './devices'
 import favorite from './favorite'
 import gregor from './gregor'
+import hmr from './hmr'
```

Comment:
sure, my reasoning was that this is separate from anything related to the user.

So I didn't want to infect user stuff with this. We could even not have this reducer when not running in dev mode